lib/miq_expression.rb

```diff
@@ -524,7 +524,22 @@ def self._to_ruby(exp, context_type, tz)
       operands = operands2rubyvalue(operator, exp[operator], context_type)
       clause = operands.join(" #{normalize_ruby_operator(operator)} ")
     when "contains"
-      operands = operands2rubyvalue(operator, exp[operator], context_type)
+      exp[operator]["tag"] ||= exp[operator]["field"]
+      operands = if context_type != "hash"
```

Comment:
> Then again, if you are attempting to explode before you compact, then LGTM

:thumbsup:

This may get pulled out at some point, but right now I'm resisting the urge to create more class methods on MiqExpression. There may be some explosions along the way. 

While this is not a good reason in support of this, worth noting not all of them are so brief - check out "FIND" for instance.